docker/grafana/dashboards/RabbitMQ-Overview.json

```diff
@@ -2067,7 +2067,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(rabbitmq_queue_messages_published{cluster=\"$cluster\"}[$__interval])) by(node)",
+          "expr": "sum(rate(rabbitmq_queue_messages_published_total{cluster=\"$cluster\"}[$__interval])) by(node)",
```

Comment:
You shouldn't have a node or cluster label to sum by, as those were coming from global_labels. It'll be (I presume) instance and cluster when that's switched to target labels.

Reply:
Yes, this will change in the second PR, when the global labels get replaced with target labels.